alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetFindingsAction.kt

```diff
@@ -90,12 +91,23 @@ class TransportGetFindingsSearchAction @Inject constructor(
 
         if (!tableProp.searchString.isNullOrBlank()) {
             queryBuilder
-                .must(
+                .should(
                     QueryBuilders
                         .queryStringQuery(tableProp.searchString)
-                        .defaultOperator(Operator.AND)
-                        .field("queries.tags")
-                        .field("queries.name")
+                )
+                .should(
+                    QueryBuilders.nestedQuery(
+                        "queries",
+                        QueryBuilders.boolQuery()
+                            .must(
+                                QueryBuilders
+                                    .queryStringQuery(tableProp.searchString)
+                                    .defaultOperator(Operator.AND)
+                                    .field("queries.tags")
+                                    .field("queries.name")
```

Comment:
Were we still planning to implement support for searching for findings using a specific `monitor_id`, or would that be better suited for a separate PR?

Reply:
The part in line 96 supports `monitor_id`, I added a integ test to showcase that